requirements.txt

```diff
@@ -1,13 +1,14 @@
 bumpversion==0.5.3
 pytest-cov>=4.0.0
 pre-commit>=2.12.0,<2.13
+pyfakefs==5.1.0
```

Comment:
Since this is the `requirements.txt` which is only used to run tests and other CI stuff (but isn't used when installing dependencies), It'd be better to pin that dependency.